### ISSUE TITLE
Install successfully but fail when require('canvas') in js

### ISSUE THREAD

**Issue by ijse** [Automattic/node-canvas#480]
```
tmp2>node index.js

module.js:356
  Module._extensions[extension](this, filename);
                               ^
Error: The specified procedure could not be found.
e:\worktop\tmp2\node_modules\canvas\build\Release\canvas.node
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (e:\worktop\tmp2\node_modules\canvas\lib\bindings.js:2:18)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)

```

Windows7 x64, 
node-v0.10.32, 
gtk+-bundle_3.6.4-20130921_win32
VS Express 2010

**Comment by PaulBGD** [comment]
@weswigham I tried both methods. While it fixed one warning in the log while building, the initial error still occurred for me.

**Comment by LinusU** [comment]
Closed in favour of #704
